package.json

```diff
@@ -140,7 +140,8 @@
     "parse-url": "^8.1.0",
     "moment-timezone": "^0.5.35",
     "**/aws-cdk-lib": "~2.68.0",
-    "**/@aws-amplify/amplify-cli-core": "^4.0.0",
+    "**/@aws-amplify/amplify-cli-core": "4.0.1",
+    "**/amplify-cli-core": "4.0.1",
```

Comment:
Why is the non-scoped version needed in resolutions?

Reply:
Probably not needed, just to be extra sure.